docs/developer/instruction/test_fixture.rst

```diff
@@ -1,5 +1,24 @@
-How to Run Tests
-================
+Description of Test Fixtures
+============================
 
 .. toctree::
    :maxdepth: 2
+
+The docker files for each test fixture is in the root of the repository with the name ``Dockerfile.<fixturename>``.
+The overall design of the test fixture is captured `here <https://code.ornl.gov/sns-hfir-scse/infrastructure/web-monitor/-/tree/next/docs/Design/Design_TestEnv>`_ .
+
+``activemq`` fixture is built from `rmohr/activemq <https://github.com/rmohr/docker-activemq>`_ with the configuration from ``src/workflow_app/workflow/icat_activemq.xml``.
+It is the activemq broker.
+
+
+``autoreducer`` fixture runs a copy of `post processing agent <https://github.com/neutrons/post_processing_agent>`_.
+This is given a fake filesystem with the contents of ``tests/data`` in the location ``/SNS/`` (at the root level of the filesystem).
+
+``amq_test_gen`` fixture creates pretend messages associated with runs being saved.
+It contains the code in the `web monitor test AMQ message generator repository <https://code.ornl.gov/sns-hfir-scse/infrastructure/web-monitor-amq-message-generator>`_ .
+
+``amq_pv_gen`` fixture creates fake process variables (PVs) that the data aquisition would make
+It contains the code in the `web monitor test pv generator repository <https://code.ornl.gov/sns-hfir-scse/infrastructure/web-monitor-test-pv-generator>`_ .
```

Comment:
same as above, can anyone view this link?  If not, should we mention how to gain access?